server/app_env/stanford_staging.py

```diff
@@ -1,22 +0,0 @@
-# Copyright 2023 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from server.app_env import _base
-
-
-class Config(_base.Config):
```

Comment:
curious why we're deleting this, because won't this code look for this file? https://github.com/datacommonsorg/website/blob/master/server/lib/config.py#L24

Reply:
In stanford_staging.yaml, the flaskEnv is now "stanford" because it should use the same html template folder (now it can not find /stanford_staging/..., thus uses the default custom html template).